samples/k8s-existing-cluster-only-sample/README.md

```diff
@@ -0,0 +1,33 @@
+---
+page_type: sample
+languages:
+- yaml
+- json
+products:
+- azure-kubernetes-service
+- azure-marketplace
+---
+
+# Existing cluster only sample
+
+This folder contains samples that shows how ISV can use only existing cluster option in their create ui definition.
+
+## The resulting UI Definition will show up as follows:
+
+![Alt text](images/UI_Sample.PNG)
+
+# Explanation
+
+For setting the scope to the existing cluster, create ui definition should store id of the existing cluster in output variable as follows
+
+## UIDefinition
+The following section creates an array of key value pairs
```

Comment:
Do we need this statement? Can this be replaced by text 'For setting the scope to the existing cluster, create ui definition should store id of the existing cluster in output variable as follows'